release/ray_release/job_manager/anyscale_job_manager.py

```diff
@@ -259,6 +264,42 @@ def run_and_wait(
         )
         return self._wait_job(timeout)
 
+    def _get_ray_error_logs(self) -> Optional[str]:
+        """
+        Obtain any ray logs that contain keywords that indicate a stack trace, such as
+        ERROR or Traceback
+        """
+        tmpdir = tempfile.TemporaryDirectory()
+        LogsController().download_logs(
```

Comment:
This LogsController is an internal concept and there is no guarantee of backwards-compatibility. I think this should work for now, but the ideal solution would involve creating a sdk command that supports this usecase well.

Reply:
should we just use launch a process and use CLI to download log?